packages/core/src/app/shipping/shippingOption/MultiShippingOptionsV2.tsx

```diff
@@ -0,0 +1,51 @@
+import { Consignment } from '@bigcommerce/checkout-sdk';
+import React from 'react';
+
+import { TranslatedString } from '@bigcommerce/checkout/locale';
+import { useCheckout } from '@bigcommerce/checkout/payment-integration-api';
+import { Alert, AlertType } from '@bigcommerce/checkout/ui';
+
+import MultiShippingOptionsListV2 from './MultiShippingOptionsListV2';
+import { isLoadingSelector } from './ShippingOptions';
+
+interface MultiShippingOptionsV2Props {
+    consignment: Consignment;
+    isLoading: boolean;
+    shippingQuoteFailedMessage: string;
+}
+
+export const MultiShippingOptionsV2 = ({
+    consignment,
+    isLoading,
+    shippingQuoteFailedMessage,
+}: MultiShippingOptionsV2Props) => {
+    const { checkoutService, checkoutState } = useCheckout();
+
+    const selectShippingOption = checkoutService.selectConsignmentShippingOption;
```

Comment:
It seems that this component is quite different from the previous shipping option, primarily because the design requires using a radio button instead of a ChecklistItem. 
As a result, this component may not be able to get anything from `./shippingOptions`, and it can be considered as a multi-shipping version of `./shippingOptions`.